README.md

```diff
@@ -45,12 +43,7 @@ An example of a denied admission would be:
 1. No valid signature or attestation was obtained for `policy2` with at least one of the `policy2` authorities
 1. The image is not admitted
 
-An example of no policy matched:
-1. If the image does not match against any policy
-1. Fallback to [deprecated policy-controller validation behavior](#deprecated-policy-controller-validation-behavior)
-1. Validation will be attempted against the secret defined under `cosign.secretKeyRef.name` during helm installation.
-  1. If a valid signature or attestation is obtained, image is admitted
-  1. If no valid signature or attestation is obtained, image is denied
+**If an image does not match a policy, it will be admitted**
```

Comment:
it will be or **not** be?
just a clarification question :)

Reply:
To be or not to be, that's the question :)

As it stands it WILL be admitted. Hence the #235 so that this behaviour can be controlled.

Reply:
Yes, let's add support for that functionality.